mmdet/models/backbones/regnet.py

```diff
@@ -204,6 +211,36 @@ def __init__(self,
         self.feat_dim = stage_widths[-1]
         self.block.expansion = expansion_bak
 
+        assert not (init_cfg and pretrained), \
+            'init_cfg and pretrained cannot be setting at the same time'
+        if isinstance(pretrained, str):
+            warnings.warn('DeprecationWarning: pretrained is a deprecated, '
+                          'please use "init_cfg" instead')
+            self.init_cfg = dict(type='Pretrained', checkpoint=pretrained)
+        elif pretrained is None:
+            if init_cfg is None:
+                self.init_cfg = [
+                    dict(type='Kaiming', layer='Conv2d'),
+                    dict(
+                        type='Constant',
+                        val=1,
+                        layer=['_BatchNorm', 'GroupNorm'])
+                ]
+            elif not isinstance(init_cfg, list):
+                self.init_cfg = [init_cfg]
+
+            # TODO: dcn conv_offset cannot initialize with init_cfg
```

Comment:
1. dcn conv_offset cannot be initialized with init_cfg
2. cannot or should not?